akka-docs-new/src/test/scala/tutorial_3/IotApp.scala

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package tutorial_3
+
+import akka.actor.ActorSystem
+import tutorial_3.DeviceManager.RequestTrackDevice
+
+import scala.io.StdIn
+
+object IotApp {
+
+  def main(args: Array[String]): Unit = {
+    val system = ActorSystem("iot-system")
+
+    try {
+      // Create top level supervisor
+      val supervisor = system.actorOf(DeviceManager.props(), "iot-supervisor")
```

Comment:
Nah, sorry, the name is right just the props is wrong. Leftover from some debug run.